openstack/nova/templates/bin/_db-online-migrate.tpl

```diff
@@ -5,8 +5,10 @@ set -e
 
 nova_manage="nova-manage --config-file /etc/nova/nova.conf"
 available_commands_text=$(nova-manage --help | awk '/Command categories/ {getline; print $0}')
+{{- if .Values.placement.enabled }}
```

Comment:
Are we able to access the release version here to check that if we're on rocky? I fear we might forget this setting until we upgrade and then wonder ...

Reply:
I am now checking for rocky in `.Values.imageVersion`  (as done in other places)